app/models/Country.scala

```diff
@@ -24,5 +24,7 @@ object Country {
 
   val GB = Country("", "GB", "United Kingdom")
 
+  val NonGB = Country("", "DE", "Germany")
```

Comment:
This shouldn't be here - rather only in the `Spec`

Reply:
@matthewharris1 - this still stands. This entry is only test related and so should be in tests and not within actual code.